ui/app/templates/components/oidc/scope-form.hbs

```diff
@@ -82,7 +82,7 @@
       <p data-test-scope-modal="text">
         Example of a JSON template for scopes:
       </p>
-      <JsonEditor @value={{this.exampleTemplate}} @mode="ruby" @readOnly={{true}} @container=".hds-modal" />
+      <Hds::CodeBlock @value={{this.exampleTemplate}} @language="ruby" @hasCopyButton={{true}} />
```

Comment:
Love getting rid of the need for `@container` - does copying data from the codeblock while in the modal still work?